xbmc/GUIInfoManager.cpp

```diff
@@ -9281,9 +9281,8 @@ void CGUIInfoManager::UpdateAVInfo()
 
       m_videoInfo = video;
       m_audioInfo = audio;
-
-      m_isPvrChannelPreview = g_PVRManager.IsChannelPreview();
     }
+    m_isPvrChannelPreview = g_PVRManager.IsChannelPreview();
```

Comment:
this is not ideal for 2 reasons

1) might bring the flickering back. updating a/v streams and preview need to be in sync.
2) it would call IsChannelPreview on every render cycle, even if the INFO is not active

How about setting IsChannelPreview to false in CGUIInfoManager::SetShowInfo if m_playerShowInfo is set to false?